cluster-autoscaler/metrics/metrics.go

```diff
@@ -207,6 +207,14 @@ var (
 		},
 	)
 
+	scaleDownEnabled = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: caNamespace,
+			Name:      "scale_down_enabled",
```

Comment:
It seems `scale_down_in_cooldown` is better name for the metric.

Reply:
Agreed, the current one makes it look like it just exposes the value of --scale-down-enabled flag or equivalent config.